Mathlib/LinearAlgebra/AffineSpace/AffineEquiv.lean

```diff
@@ -15,64 +15,72 @@ import Mathlib.Algebra.Invertible
 /-!
 # Affine equivalences
 
-In this file we define `affine_equiv k P₁ P₂` (notation: `P₁ ≃ᵃ[k] P₂`) to be the type of affine
+In this file we define `AffineEquiv k P₁ P₂` (notation: `P₁ ≃ᵃ[k] P₂`) to be the type of affine
 equivalences between `P₁` and `P₂, i.e., equivalences such that both forward and inverse maps are
 affine maps.
 
 We define the following equivalences:
 
-* `affine_equiv.refl k P`: the identity map as an `affine_equiv`;
+* `AffineEquiv.refl k P`: the identity map as an `AffineEquiv`;
 
-* `e.symm`: the inverse map of an `affine_equiv` as an `affine_equiv`;
+* `e.symm`: the inverse map of an `AffineEquiv` as an `AffineEquiv`;
 
-* `e.trans e'`: composition of two `affine_equiv`s; note that the order follows `mathlib`'s
-  `category_theory` convention (apply `e`, then `e'`), not the convention used in function
+* `e.trans e'`: composition of two `AffineEquiv`s; note that the order follows `mathlib`'s
+  `CategoryTheory` convention (apply `e`, then `e'`), not the convention used in function
   composition and compositions of bundled morphisms.
 
-We equip `affine_equiv k P P` with a `group` structure with multiplication corresponding to
-composition in `affine_equiv.group`.
+We equip `AffineEquiv k P P` with a `Group` structure with multiplication corresponding to
+composition in `AffineEquiv.group`.
 
 ## Tags
 
 affine space, affine equivalence
 -/
 
-
 open Function Set
 
 open Affine
 
+section defn
+
+variable (k : Type _) [Ring k]
```

Comment:
Can you put this back in the order it originally was? This will make `@AffineEquiv` not align correctly, and also isn't the usual order of arguments that we prefer.